From e1e6e474404365587202d1a1dbfc13f4ac41d2e9 Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Mon, 31 Oct 2022 08:08:19 +0000
Subject: [PATCH] Add isVisible public API.

Task visibilty info will help us decide if correspoding
LG window should be closed or not.

This patch also adds TASK permission to ClipboardAgent
private app.

Tracked-On: OAM-104429
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 data/etc/privapp-permissions-platform.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 8c820f89fda4..aa3a1b819064 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -555,6 +555,7 @@ applications that come with the platform
         <permission name="android.permission.INTERNAL_SYSTEM_WINDOW"/>
         <permission name="android.permission.INTERNET"/>
         <permission name="android.permission.PACKAGE_USAGE_STATS"/>
+        <permission name="android.permission.GET_TASKS"/>
     </privapp-permissions>
     
 </permissions>
-- 
2.17.1

